april-2021/looking-to-the-future.md

```diff
@@ -0,0 +1,48 @@
+We’ve also added some other quality of life enhancements behind the scenes, such as [some automated tools to check PHP syntax for all Pull Requests and Commits](https://github.com/mybb/mybb/pull/4200) to the GitHub repository and some improvements to our support for PHP 8.0.
+
+The release of 1.8.27 has been a long time coming, but it’s now just around the corner. I’d like to take this opportunity to thank [all of our wonderful contributors](https://github.com/mybb/mybb/graphs/contributors), and to ask a favour: if you can, please test the current code as much as you can! We want to make sure that 1.8.27 is a rock solid release.
+
+## The future of 1.8
+
+With 1.8.27 being such a large release, we’ve been thinking about the future of where MyBB is headed.
+
+As such, we’ve decided that **MyBB 1.8.27 will be the last major release of MyBB 1.8**.
+
+From this point forwards, the 1.8 series shall only receive security fixes and bug fixes for critical bugs that break core functionality.
+
+The reasoning for this change is simple: we need to focus all of our limited resources on one single task - namely, getting MyBB 1.9 released.
+
+## MyBB 1.9
+
+MyBB 1.9 is something we’ve been talking about for a long time (too long, a lot of us would say).
+
+We’ve been working on it side-by-side along with supporting the MyBB 1.8 series, which has unfortunately meant the new release has only had limited attention paid to it.
+
+With 1.8.27 being the last big 1.8 series release, all attention will now be turned to 1.9. There will be a freeze made to the merging of any Pull Requests to the project for a period of roughly two weeks in order to finalise the rebase of MyBB 1.9 to incorporate all of the changes present in 1.8.27.
+
+Once this is complete, attention will turn to the following tasks:
+
+* Scrutinising all new templates to ensure that all changes made to 1.8 in recent releases are reflected in the new templates.
+* Tracking down any remaining usages of the old `$templates` based code for templates.
+* Writing the ACP management module for the new template system.
+
+Once these tasks are complete, we’ll be at the stage of beginning testing the release in full. At that point, we’ll put a demo install online for everybody to play with, which will reset every day at midnight. This should give everybody a chance to help us debug the release and polish it up.
+
+## An apology and a thanks
+
+On a final personal note, I’d like to apologise to the community for the severe lack of progress with the project and communication from us.
+
+When I joined the team, forums were booming and MyBB in particular was abuzz with activity. We had a large bustling team with members from all over the world contributing many changes and improvements. I’ve watched the project go from MyBB 1.2 to 1.4; from 1.4 to 1.6 and 1.6 to 1.8. Over that time, things have changed a lot! The rise of Social Media and Smartphones have changed the landscape of internet communities significantly.
```

Comment:
I don't think _social media_ and _smartphones_ don't need to be capitalized. probably _Internet_.